python/doc/examples/reliability_sensitivity/reliability/plot_nais.py

```diff
@@ -0,0 +1,237 @@
+"""
+Non parametric Adaptive Importance Sampling (NAIS)
+==================================================
+"""
+# %%
+#
+# The objective is to evaluate a probability from the Non parametric Adaptive Importance Sampling (NAIS) technique.
+#
+# We consider the four-branch function :math:`g : \mathbb{R}^2 \rightarrow \mathbb{R}` defined by:
+#
+# .. math::
+#   \begin{align*}
+#   g(X) = \min \begin{pmatrix}5+0.1(x_1-x_2)^2-\frac{(x_1+x_2)}{\sqrt{2}}\\
+#   5+0.1(x_1-x_2)^2+\frac{(x_1+x_2)}{\sqrt{2}}\\
+#   (x_1-x_2)+ \frac{9}{\sqrt{2}}\\
+#   (x_2-x_1)+ \frac{9}{\sqrt{2}}
+#   \end{pmatrix}
+#   \end{align*}
+#
+# and the input random vector :math:`X = (X_1, X_2)` which follows a Normal distribution with independent components, and identical marginals with zero mean and unit variance:
+#
+# .. math::
+#   \begin{align*}
+#   X \sim  \mathcal{N}(\mu = [0, 0], \sigma = [1,1], corr = I_2)
+#   \end{align*}
+#
+# We want to evaluate the probability:
+#
+# .. math::
+#   \begin{align*}
+#   p = \mathbb{P} \( g(X) \leq 0 \)
```

Comment:
```suggestion
#   p = \mathbb{P} ( g(X) \leq 0 )
```
I assume this is why the example does not compile.

![image](https://user-images.githubusercontent.com/31989332/188477272-f75a1dd0-79ac-4571-9336-adb888383c50.png)